weave/trace_server/trace_server_interface.py

```diff
@@ -20,36 +65,41 @@ class CallSchema(BaseModel):
     # Optional display name of the call
     display_name: typing.Optional[str] = None
 
-    ## Trace ID
+    # Trace ID
     trace_id: str
-    ## Parent ID is optional because the call may be a root
+    # Parent ID is optional because the call may be a root
     parent_id: typing.Optional[str] = None
 
-    ## Start time is required
+    # Start time is required
     started_at: datetime.datetime
-    ## Attributes: properties of the call
-    attributes: typing.Dict[str, typing.Any]
+    # Attributes: properties of the call
+    attributes: AttributeMap
 
-    ## Inputs
+    # Inputs
     inputs: typing.Dict[str, typing.Any]
 
-    ## End time is required if finished
+    # End time is required if finished
     ended_at: typing.Optional[datetime.datetime] = None
 
-    ## Exception is present if the call failed
+    # Exception is present if the call failed
     exception: typing.Optional[str] = None
 
-    ## Outputs
+    # Outputs
     output: typing.Optional[typing.Any] = None
 
-    ## Summary: a summary of the call
-    summary: typing.Optional[typing.Dict[str, typing.Any]] = None
+    # Summary: a summary of the call
+    summary: typing.Optional[SummaryMap] = None
 
     # WB Metadata
     wb_user_id: typing.Optional[str] = None
     wb_run_id: typing.Optional[str] = None
 
     deleted_at: typing.Optional[datetime.datetime] = None
+    latency: typing.Optional[int] = None
+
+    @field_serializer("attributes", "summary", when_used="unless-none")
+    def serialize_typed_dicts(self, v):
+        return dict(v)
```

Comment:
retains extra keys (not typed in the type dict) when `call.model_dump()` cc @andrewtruong  in case you are interested